packages/cli/generators/discover/index.js

```diff
@@ -0,0 +1,279 @@
+path = require('path');
+const fs = require('fs');
+const ArtifactGenerator = require('../../lib/artifact-generator');
+const modelMaker = require('../../lib/model-discoverer');
+const debug = require('../../lib/debug')('discover-generator');
+const utils = require('../../lib/utils');
+const modelDiscoverer = require('../../lib/model-discoverer');
+
+module.exports = class DiscoveryGenerator extends ArtifactGenerator {
+  constructor(args, opts) {
+    super(args, opts);
+
+    this.option('dataSource', {
+      type: String,
+      alias: 'ds',
+      description: 'The name of the datasource to discover',
+    });
+
+    this.option('views', {
+      type: Boolean,
+      description: 'Boolean to discover views',
+      default: true,
+    });
+
+    this.option('schema', {
+      type: String,
+      description: 'Schema to discover',
+      default: '',
+    });
+
+    this.option('all', {
+      type: Boolean,
+      description: 'Discover all models without prompting users to select',
+      default: false,
+    });
+
+    this.option('schemaDefs', {
+      type: String,
+      description:
+        'The path to a .json file with an array of schema definition',
+      default: undefined,
+    });
+  }
```

Comment:
I don't see `--outDir` defined in generator options. It's defined in the docs though. What am I missing?

Reply:
Woops, fixed